wire-ios/Wire-iOS.xcodeproj/project.pbxproj

```diff
@@ -8352,7 +8352,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "# Adds support for Apple Silicon brew directory\nexport PATH=\"$PATH:/opt/homebrew/bin\"\n \nif [ -z \"$CI\" ]; then\n  if which swiftlint >/dev/null; then\n    swiftlint\n  else\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\n  fi\nelse\n  echo \"Skipping SwiftLint in CI environment\"\nfi\n";
+			shellScript = "# Adds support for Apple Silicon brew directory\nexport PATH=\"$PATH:/opt/homebrew/bin\"\n \nif [ -z \"$CI\" ]; then\n  if which swiftlint >/dev/null; then\n    swiftlint --config ../.swiftlint.yml\n  else\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\n  fi\nelse\n  echo \"Skipping SwiftLint in CI environment\"\nfi\n";
```

Comment:
what about moving all this duplicate script to a single bash script under `scripts/`